src/main/java/teammates/ui/controller/FeedbackSubmissionEditSaveAction.java

```diff
@@ -239,7 +241,7 @@ private boolean isExistingResponseValid(FeedbackResponseAttributes response) {
         return existingResponsesId.contains(response.getId());
     }
 
-    private void saveResponse(FeedbackResponseAttributes response)
+    private void saveResponse(FeedbackResponseAttributes response, List<FeedbackResponseAttributes> newResponses)
```

Comment:
Naming issue - it's a little weird that now `saveResponse` doesn't actually save the response for new responses, but collects them. Could we call it `collectResponse` instead? Maybe the responses to be deleted and updated can be collected in separate lists too, then just use for loops to go through them in another method? To be consistent.